docs/tutorial.md

```diff
@@ -0,0 +1,506 @@
+
+<img width="537" alt="screen shot 2019-01-08 at 15 32 15" src="https://user-images.githubusercontent.com/42912128/50833654-9b008e00-135a-11e9-9e08-f1d51f1dc52a.png"></img>
+
+
+>I want this playbook to translate the details field in an incident into yoda speak, so for “text” we will click the brackets right here. 
+
+![brackets](https://user-images.githubusercontent.com/42912128/50833717-d69b5800-135a-11e9-85f5-81b6a1dd1e8c.png)
+
+>Next select incident details and click on “Details”.
+
+![details](https://user-images.githubusercontent.com/42912128/50833826-24b05b80-135b-11e9-8d18-3ee1d52301c1.png)
+
+>Go ahead and click “Close” and “OK”.
+
+This saves your changes. If you do not click close and okay, your changes will not be saved in the playbook.
+
+>Now let’s have the playbook print the translation in the war room. In the task library search for “print” under “Utilities”.
+
+![print_add](https://user-images.githubusercontent.com/42912128/50833982-8b357980-135b-11e9-9d0e-0d85834e3c65.png)
+
+>For the value click the brackets. Notice how we have an entry here for the yoda speak translation? If we click it, we can select the translation context that we specified in the integration settings.
+
+![print_settings](https://user-images.githubusercontent.com/42912128/50834106-e1a2b800-135b-11e9-892f-a00c74b949f6.png)
+
+>Click “Close” and “OK”. Lastly, hit the save button and click the “X”.
+
+Again we must commit our changes to the playbook.
+
+We need to connect the two tasks together. Do so by dragging an arrow from the bottom of the translate task to the top of the print task.
+
+![connect](https://user-images.githubusercontent.com/42912128/50840624-94c6dd80-136b-11e9-832e-0a745b2d754a.png)
+
+
+>Let’s see it in action. Click “Incidents” and then press the blue button that says “New Incident”.
+
+![incidents](https://user-images.githubusercontent.com/42912128/50834260-3cd4aa80-135c-11e9-80f7-39d80d55d7e9.png)
+
+>I’m going to name this “That’s no moon… It’s a Space station!” and for the details, lets type “The prequel movies are more entertaining than the new Disney movies”. 
+
+You also need to select which playbook to attach to the incident. In this case, we would attach the "yodaspeak" playbook.
+
+<img width="1350" alt="screen shot 2019-01-08 at 15 46 39" src="https://user-images.githubusercontent.com/42912128/50834429-a6ed4f80-135c-11e9-9029-40c130be7371.png"></img>
+
+>Click “Create new Incident” and select the incident we just created. Navigate to “work plan” and we can see that our playbook worked!
+
+![screen shot 2019-01-08 at 16 09 53](https://user-images.githubusercontent.com/42912128/50835627-e49fa780-135f-11e9-8c96-8c3e138d6a9b.png)
+
+## Resources
+
+* [YodaSpeak Integration](https://github.com/demisto/content-docs/tree/master/tutorial/integration-Yoda_Speak.yml)
+* [YodaSpeak Playbook](https://github.com/demisto/content-docs/tree/master/tutorial/playbook-Yoda_Speak.yml)
```

Comment:
These links do not work even if the files are present in /tutorial 

@sserrata do we need them to be in /docs ? Or can we add them to /static/tutorial folder?

Reply:
The links will work only after we merge into master. I wanted to reference the source files so it will be easy to review and download.